src/aoclient.cpp

```diff
@@ -264,10 +264,16 @@ void AOClient::sendServerBroadcast(QString message)
 bool AOClient::checkAuth(unsigned long long acl_mask)
 {
     if (acl_mask != ACLFlags.value("NONE")) {
-        if (!authenticated) {
+        if (!authenticated && acl_mask != ACLFlags.value("CM")) {
             return false;
         }
 
+        if (acl_mask == ACLFlags.value("CM")) {
+            AreaData* area = server->areas[current_area];
+            if (area->owners.contains(id))
+                return true;
+        }
+
```

Comment:
if you move this if statement before the `if (!authenticated && ...`, you don't need to check if the acl_mask is equal to ACLFlags.value("CM")